docs/source/index.rst

```diff
@@ -35,6 +35,14 @@ Contents:
 Upgrade Notes
 =============
 
+Upgrading to 1.0.0rc1
```

Comment:
I forgot that we started this. There may be some things that I forgot to add like the update to the pagination interface, but I am not entirely sure when it was added.